packages/client/modules/demo/components/CreateAccountSection.tsx

```diff
@@ -124,6 +124,7 @@ const CreateAccountSection = (props) => {
         <td style={buttonCellStyle}>
           <a
             data-cy='create-account'
```

Comment:
As a separate PR, I'd like to run through and clean these `data-cy` properties up. Ideally, we only use HTML elements/text/aria to navigate the page when possible.